deis-dev/manifests/deis-minio-rc.yaml

```diff
@@ -18,8 +18,24 @@ spec:
         - name: deis-minio
           image: quay.io/deisci/minio:v2-beta
           imagePullPolicy: Always
+          env:
+            - name: HEALTH_SERVER_PORT
+              value: "8082"
           ports:
             - containerPort: 9000
+            - containerPort: 8082
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: 8092
+            initialDelaySeconds: 30
+            timeoutSeconds: 1
```

Comment:
> if that takes longer than 1 second, the service should be considered down

Makes sense--there probably isn't a "one size fits all" value to use everywhere. :+1: